roles/imageset_mirroring/tasks/main.yml

```diff
@@ -0,0 +1,50 @@
+---
+- name: Mirror operators using oc-mirror
+  block:
+
+    - name: Validate im_task parameter
+      ansible.builtin.assert:
+        that:
+          - im_task is defined
+          - im_task in ['latest', 'all', 'custom']
+        fail_msg: "im_task must be set to one of: 'latest', 'all', 'custom'."
+        success_msg: "Valid mirror task selected: {{ im_task }}"
```

Comment:
Not needed any longer, that's the nice thing about argument_specs, those are validated before entering the task file. ;)